### PR TITLE
chore: move unit test from travis ci to github actions

### PR DESCRIPTION
**Pull request by kakhaUrigashvili**
Moving unit test ci build to github actions

Few reasons
1) Seems to run faster
2) Github actions can help us better organize some other automated checks. For example, we will add intergration-test.yml in .github/workflows folder in the integration test PR.

### CODE REVIEW

File: .github/workflows/unit-test.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        node: [8, 10, 12]
```
Comment (kakhaUrigashvili) [inline .github/workflows/unit-test.yml:12]: Only difference you can not specify latest stable node version, but the feature is coming: https://github.com/actions/setup-node/pull/58

File: .github/workflows/unit-test.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node }}
+    - run: npm install
+    - run: npm run lint
+    - run: npm run test:report
```
Comment (RonWang) [inline .github/workflows/unit-test.yml:22]: no new line at end of file

File: .travis.yml [hunk 3]
```diff
@@ -1,27 +0,0 @@
-language: node_js
```
Comment (RonWang) [inline .travis.yml:1]: README is using a badge to track travis status. Does that have equivalence?

Comment (kakhaUrigashvili) [reply .travis.yml:1]: Yes. added

File: .github/workflows/unit-test.yml (hunk 2 above)
Comment (kakhaUrigashvili) [reply .github/workflows/unit-test.yml]: added

**Pull request merged**